matlab_simulation/02-examples_practice/gyroCovEstimation/gyroCovEstimation.m

```diff
@@ -0,0 +1,90 @@
+%% A Matlab program to calculate the convariances from a Gyroscope Data.
+% It demonstrates the following aspects:
+% ----Contributions in Ver.2.0
+% -Vectorized the Gyro Data.
+% -More Compact Code.
+% -Removed repeatations
+% -Completed the Error Distribution Comparison at the bottom.
+% -Commented wherever possible
+% 
+% Author: Bismaya Sahoo
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Initializations
+clear all; close all; clc;
+RAD2DEG=180/pi;
+
+tmin = 10; % Put Start_time here in seconds
+tmax = 20; % Put Stop_time here in seconds
+
+%% Load Data and Initial Calculations
+load('gyroData.mat'); % Load the Gyro Dataset.
```

Comment:
I ran the "MobileRoboticsSetup.m" and then ran this file. It produced the following error:
Error using load, Unable to read file 'gyroData.mat': no such file or directory.
I would suggest either placing the .mat file in the same directory as the example or updating the "MobileRoboticsSetup.m" file to include the new directory you created.